.travis.yml

```diff
@@ -1,4 +1,19 @@
 language: go
 
 go:
-  - 1.13.x
\ No newline at end of file
+  - 1.13.x
+
+services:
+  - docker
+
+script:
+  - make
+
+before_deploy:
+  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+
+deploy:
+  provider: script
+  script:  make docker-push
+  on:
+    branch: master
```

Comment:
does the lack of trailing EOL cause any problems?

Reply:
I think it shouldn't cause any issues. We can merge and see )